packages/react-reconciler/src/ReactFiberBeginWork.old.js

```diff
@@ -2808,7 +2826,9 @@ function updateDehydratedSuspenseComponent(
 
       // If we have scheduled higher pri work above, this will just abort the render
       // since we now have higher priority work. We'll try to infinitely suspend until
-      // we yield. TODO: We could probably just force yielding earlier instead.
+      // we yield.
+      // TODO: This should only happen for sync updates, which don't have a
+      // hydration lane. We should add one. Then, consider removing this call.
       renderDidSuspendDelayIfPossible();
```

Comment:
Is this call just unnecessary now? This flag doesn't do anything in sync mode anyway?